### PR TITLE
Add Web Portals to platform portal

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Added WebPortal entity into platform portal.

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/platform/src/router/EntityMap.ts [hunk 1]
```diff
@@ -29,6 +29,14 @@ const getEntityMap = (): ExtendedRouteMap => {
         },
       ],
     },
+    {
+      label: _('Where I go? ¯  _ (o_o) _ / ¯'),
+      children: [
+        {
+          entity: entities.WebPortal,
```
Comment (mmadariaga) [inline web/portal/platform/src/router/EntityMap.ts:36]: Move entity into parent block, and remove this label

File: web/portal/platform/src/entities/WebPortal/WebPortal.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    label: _('Logo'),
+    type: 'file',
+  },
+};
+
+const WebPortal: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: InsertLinkIcon,
+  iden: 'WebPortal',
+  title: _('Company Portal', { count: 2 }),
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/WebPortal/WebPortal.tsx:84]:
```suggestion
  title: _('Platform Portal', { count: 2 }),
```

File: web/portal/platform/src/entities/WebPortal/WebPortal.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    },
+  },
+  name: {
+    label: _('Name'),
+    maxLength: 200,
+    helpText: _(`Will be shown on page footer`),
+  },
+  userTheme: {
+    label: _('User Theme'),
+  },
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/WebPortal/WebPortal.tsx:73]: I this taht userTheme is not needed in platform portal

File: web/portal/platform/src/entities/WebPortal/WebPortal.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+};
+
+const WebPortal: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: InsertLinkIcon,
+  iden: 'WebPortal',
+  title: _('Company Portal', { count: 2 }),
+  path: '/web_portals',
+  toStr: (row: any) => row.id,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/WebPortal/WebPortal.tsx:86]:
```suggestion
  toStr: (row: any) => row.name,
```

File: web/portal/platform/src/entities/WebPortal/ForeignKeyResolver.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { autoForeignKeyResolver } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { foreignKeyResolverType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { WebPortalPropertiesList } from './WebPortalProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
+const foreignKeyResolver: foreignKeyResolverType = async function ({
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/WebPortal/ForeignKeyResolver.tsx:6]: We don't need this file

File: web/portal/platform/src/entities/WebPortal/ForeignKeyGetter.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { WebPortalPropertyList } from './WebPortalProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
+export const foreignKeyGetter: ForeignKeyGetterType = async ({
+  cancelToken,
+  entityService,
+}): Promise<any> => {
+  const response: WebPortalPropertyList<unknown> = {};
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/WebPortal/ForeignKeyGetter.tsx:10]: We don't need this file

**Review by mmadariaga — APPROVED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**